rust/kcl-lib/Cargo.toml

```diff
@@ -50,6 +50,7 @@ lazy_static = { workspace = true }
 measurements = "0.11.0"
 miette = { workspace = true }
 mime_guess = "2.0.5"
+nalgebra-glm = "0.19.0"
```

Comment:
I learned that this is in the same repo as nalgebra, so the concerns I had don't apply. 👍